### ISSUE TITLE
Uncaught SyntaxError: await is only valid in async function

### ISSUE THREAD

**Issue by JoeSpencer** [blitz-js/blitz#1306]
### Steps to Reproduce

1. In Blitz console, 'let q = await db.question.create({data: {text: "What's new?}})'

### Versions

```
debug: blitzPkgPath: /Users/jspencer/Code/BlitzTest/node_modules/blitz/dist/index.js
debug: cliPkgPath: /Users/jspencer/Code/BlitzTest/node_modules/@blitzjs/cli/lib/src/index.js 

macOS Catalina | darwin-x64 | Node: v12.18.4

blitz: 0.24.1 (global)
blitz: 0.24.1 (local)

  Package manager: yarn 
  System:
    OS: macOS 10.15.6
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 2.62 GB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 12.18.4 - /usr/local/bin/node
    Yarn: 1.22.4 - /usr/local/bin/yarn
    npm: 6.14.6 - /usr/local/bin/npm
    Watchman: Not Found
  npmPackages:
    @prisma/cli: 2.8.1 => 2.8.1 
    @prisma/client: 2.8.1 => 2.8.1 
    blitz: 0.24.1 => 0.24.1 
    react: 0.0.0-experimental-7f28234f8 => 0.0.0-experimental-7f28234f8 
    react-dom: 0.0.0-experimental-7f28234f8 => 0.0.0-experimental-7f28234f8 
    typescript: 4.0.3 => 4.0.3 
```

### Other

![Screen Shot 2020-10-12 at 4 49 25 PM](https://user-images.githubusercontent.com/3081871/95793774-f2f4c300-0cab-11eb-9010-61fc9c93c632.png)

**Comment by flybayer** [comment]
Thanks @JoeSpencer. As said in Slack, I am unable to reproduce. Hopefully someone who can reproduce this can investigate what's broken.

**Comment by wKovacs64** [comment]
@JoeSpencer is the example in your screenshot a specific command you tried, verbatim? If so, you're missing a closing quote between the `?` and the `}` towards the end.

You have:

```
await db.question.create({data: {text: "What's new?}})
```

Should be:

```
await db.question.create({data: {text: "What's new?"}})
```

I was unable to reproduce with valid commands/syntax.

**Comment by JoeSpencer** [comment]
So that was the problem, I am closing this, but weird error to trigger for that, no?

**Comment by renet** [comment]
I ran into the same problem without a syntax error, following the tutorial, but with one exception: I didn't install blitz globally, but was using `npx blitz console` instead. This was actually causing the problem. Queries like `await db.question.findMany()` didn't work with npx, but worked just fine as soon as I used a globally installed blitz.

(Off topic: Running `npx blitz new <project-name>` in the first place prompted me to install blitz, which caused a new node_modules folder in my development directory. But otherwise worked fine until I tried to use the blitz console. Are you planning to support `npx blitz` in the future?)
